src/client/pages/OfferNew/utils.ts

```diff
@@ -34,6 +34,9 @@ export const getOfferData = (quoteBundle: QuoteBundle): OfferData => {
     },
     quotes: quoteBundle.quotes.map((bundleQuote) => {
       return {
+        firstName: bundleQuote.firstName,
+        lastName: bundleQuote.lastName,
+        birthDate: bundleQuote.birthDate,
```

Comment:
Do we need the `firstName`, `lastName` and the `birthDate` here when we have them on the `person` object? I mean they won't differ between the quotes? Or what did I miss? 😄

Reply:
You are correct!